main.py

```diff
@@ -2,6 +2,8 @@
 
 from play_button import PlayButton
 
+from menu import MainMenu
```

Comment:
Нема потреби в пістій стрічці тут
```suggestion
from menu import MainMenu
```